pkgs/by-name/fi/firefly-iii/package.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenvNoCC
+, lib
+, fetchzip
+, php83
```

Comment:
I'm wondering whether to change this to `phpPackage ? php83` and make this overridable. Thoughts?